src/content/docs/vulnerability-management/dev-workflow.mdx

```diff
@@ -26,7 +26,7 @@ Vulnerability data sent through [one of our integrations](/docs/vulnerability-ma
 
 Once vulnerability data starts flowing into New Relic, you can access your data through various scoped views. 
 
-To monitor the health of specific applications or services, use our entity scoped view by navigating to **[newrelic.com](https://newrelic.com/all-capabilities) > APM and services** > select an entity > then select **Security** under the **Triage** tab. For a larger scope, see our document on [managing vulnerabilities as a security team](/docs/vulnerability-management/security-workflow).
+To monitor the health of specific applications or services, use our entity scoped view by navigating to **[one.newrelic.com](https://newrelic.com/all-capabilities) > APM and services > select an entity > then select Security under the Triage** tab. For a larger scope, see our document on [managing vulnerabilities as a security team](/docs/vulnerability-management/security-workflow).
```

Comment:
This path still seems off to me. Generally the bold is reserved only for thing the user does, or things in the middle of a path in a parentheses. It should also begin with `go to`. Should probably be something like one of these:

```...scoped view. Go to **[one.newrelic.com](https://newrelic.com/all-capabilities) > APM and services**. Then select an entity, then select **Security** under the **Triage** tab.```

or

```...scoped view. Go to **[one.newrelic.com](https://newrelic.com/all-capabilities) > APM and services > (select an entity) > Triage > Security**.```

Also, can we avoid "entity" here in favor of the more human "app"?